apps/wallet/src/generated/station/station.did

```diff
@@ -1567,6 +1603,17 @@ type ChangeCanisterResourceAction = variant {
   Create;
 };
 
+// The target canister of a request to change a managed canister.
+type ChangeManagedCanisterResourceTarget = variant {
+  Any;
+  Canister : principal;
+};
+
+// The actions that are available for managed canister changes.
+type ChangeManagedCanisterResourceAction = variant {
+  Create : ChangeManagedCanisterResourceTarget;
```

Comment:
What do you think of changing the approach here a bit?since we are adding a separate operation anyway and we will need to do operations like Create the canister for the first time and then also Manage it, what do you think of

```
type ManagedCanisterResourceAction = variant {
  Create;
  Change : ManagedCanisterResourceTarget;
}
```

This way we would later be able to expand this with something like `Call`, etc... and the operations for those would remain separate but mapping to the expective action, e.g.

```
ManagedCanisterResourceAction::Create -> CreateManagedCanisterOperation
ManagedCanisterResourceAction::Change -> ChangeManagedCanisterOperation
ManagedCanisterResourceAction::Call -> CallManagedCanisterOperation
```